packages/webapp/src/types.ts

```diff
@@ -122,6 +122,7 @@ export enum AuthModes {
 
 export interface Connection {
     id: number;
+    connection_id: string;
```

Comment:
😒

Reply:
why do we need it though? I am sure we are eventually going to run into bugs because of it

Reply:
Depending on the API it sometimes sends `connectionId` and sometimes `connection_id`. I agree it should be fixed but unsure if it is in the scope of this PR. Let me know if you think it is a blocker for this PR @bodinsamuel @TBonnin

Reply:
up to you @khaliqgant  but I would prefer if we fix this as soon as possible

Reply:
FYI updated with https://github.com/NangoHQ/nango/pull/1993/commits/986a22b09d21b8749d2fb9773ed6b610831fafa9